other/materials_designer/create_nanowire_custom_shape.ipynb

```diff
@@ -0,0 +1,273 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.    VACUUM = 10.0 # Vacuum thickness on the sides in Angstrom
</pre><p>in Angstroms</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/162/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>